src/bokeh/embed/bundle.py

```diff
@@ -145,46 +145,28 @@ def add(self, artifact: Artifact) -> None:
         elif isinstance(artifact, Style):
             self.css_raw.append(artifact.content)
 
-def bundle_for_objs_and_resources(objs: Sequence[Model | Document] | None,
-        resources: BaseResources | tuple[BaseResources | None, BaseResources | None] | None) -> Bundle:
+def bundle_for_objs_and_resources(objs: Sequence[Model | Document] | None, resources: Resources | None) -> Bundle:
     ''' Generate rendered CSS and JS resources suitable for the given
     collection of Bokeh objects
 
     Args:
         objs (seq[Model or Document]) :
 
-        resources (BaseResources or tuple[BaseResources])
+        resources (Resources)
 
     Returns:
         Bundle
 
     '''
-    # Any env vars will overide a local default passed in
-    resources = settings.resources(default=resources)
```

Comment:
Just double checking, this won't prevent e.g. 
```
BOKEH_RESOURCES=inline python foo.py
```
correct? I use that constantly and have advised others on it many many times. I'm guessing the codepaths in `bokeh/resources.py` take care of this, but it would be nice to confirm (and it which case decoupling the settings from here is terrific)